### ISSUE TITLE
Admin UI login problem.

### ISSUE THREAD

**Issue by Loiuy123** [telekom-security/tpotce#989]
Hello there.

Just installed tpotce and I am trying to login to Admin UI but password or username are not correct for some reason.

When I login to Web UI everything works, but when I try to login to Admin Ui providing the same username and passoword it doesn't work.

**Comment by t3chn0m4g3** [comment]
Yes, because that is a different set of username and password. Please consult the ReadMe for the differences.

**Comment by Loiuy123** [comment]
Yeah, I've read it but I still don't understand what my username or password is. I am using the one that I typed in during the installation but it won't work. 

And also there is a problem when I try to go to Kibana : `stausCode":503,"error":"Service Unavailable","message":"License is not available."}`

**Comment by Loiuy123** [comment]
Well, I've read it.

Tried multiple things but still can't log in.

And there is also another problem with kibana: `{"statusCode":503,"error":"Service Unavailable","message":"License is not available."}`

**Comment by Loiuy123** [comment]
Resolved with the help of this comment: https://github.com/telekom-security/tpotce/issues/350#issuecomment-482831562
